ehitchhiking-ui/src/app/shared/services/api.services/black-list-api.service.ts

```diff
@@ -0,0 +1,47 @@
+import {User} from '@shared/models/user';
+import {ApiService} from '@shared/services/api.services/api.service';
+import {Car} from '@shared/models/car';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+import {map} from 'rxjs/operators';
+import {Injectable} from '@angular/core';
+
+@Injectable()
+export class BlackListApiService {
+	constructor(private apiService: ApiService) {}
+
+	mapBlackListPassengers(data): User[] {
+		return data.map((obj) => {
+			return new User(obj.id, obj.firstName + ' ' + obj.lastName, '', obj.email, '', [
+				new Car('ferrari', 'pink', 'A3434B', 1),
+			]);
+		});
+	}
+
+	mapBlacklistDrivers(data): User[] {
+		return data.map((obj) => {
+			return new User(obj.id, obj.firstName + ' ' + obj.lastName, '', obj.email, '', [
+				new Car('ferrari', 'pink', 'A3434B', 1),
+			]);
+		});
+	}
+
+	getDriverBlacklist(params: GetBlockedPassengersParams) {
```

Comment:
GetBlockedPassengersParams and GetBlockedDriversParams are equal. Pls leave one of them and rename to GetBlockedUsersParams

Reply:
done